src/Entity/Capture.php

```diff
@@ -0,0 +1,926 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Acquia\LiftClient\Exception\LiftSdkException;
+use Acquia\LiftClient\Utility\Utility;
+use DateTime;
+
+class Capture extends Entity
+{
+    /**
+     * Set the 'person_udf$' field.
+     *
+     * @param int    $num   Which field to fetch. Can be from 1 till 50
+     * @param string $value Custom fields for the Acquia Lift Person Capture Phase
+     *
+     * @throws \Acquia\LiftClient\Exception\LiftSdkException
+     *
+     * @return \Acquia\LiftClient\Entity\Capture
+     */
+    public function setPersonUdf($num, $value)
+    {
+        if (!is_int($num)) {
+            throw new LiftSdkException('Argument must be an instance of integer.');
+        }
+        if ($num < 1 || $num > 50) {
```

Comment:
I would personally define a static variable to contain the weird 50, 20, 50 UDF counts in one place. Lift Web has changed the counts before (as per client requests), and is likely to change again sometime. It could save us trouble to centralize the counts definition in each repo.